test/extensions/filters/http/common/fuzz/uber_per_filter.cc

```diff
@@ -70,12 +71,27 @@ void UberFilterFuzzer::guideAnyProtoType(test::fuzz::HttpData* mutable_data, uin
   mutable_any->set_type_url(type_url);
 }
 
+void removeConnectMatcher(Protobuf::Message* message) {
+  envoy::extensions::filters::http::jwt_authn::v3::JwtAuthentication& config =
+      dynamic_cast<envoy::extensions::filters::http::jwt_authn::v3::JwtAuthentication&>(*message);
+  for (auto& rules : *config.mutable_rules()) {
```

Comment:
Does this one work as an API constraint as well?

Reply:
Nope because connect matcher is supported elsewhere.
I pinged slack channel and tagged @qiwzhang because I wasn't sure if it should be supported (this is the only remaining usecase unsupported)

I could do a workaround and define the constraint in the route proto, and ignore embedded message validation for the places that do support connect match, but I'm not certain it'll work (as in, other validations will be ignored)

Reply:
Seems reasonable. Can you also add a TODO in JWT authenticator filter to fix the fuzzer when the implementation happens?